src/pulse3D/metrics.py

```diff
@@ -154,22 +153,28 @@ def calculate_amplitudes(
         Returns:
             Pandas Series of float values representing the amplitude of each twitch
         """
+        twitch_indices_hashable = HashableDataFrame(DataFrame.from_dict(twitch_indices).T)
+        filtered_data_hashable = tuple(tuple(d) for d in filtered_data)
+        _, coordinates = TwitchWidth.calculate_twitch_widths(
+            twitch_indices=twitch_indices_hashable,
+            filtered_data=filtered_data_hashable,
+            rounded=rounded,
+            twitch_width_percents=(10, 90),
+        )
         data_series = filtered_data[1, :]
 
         estimates_dict: Dict[int, float] = dict()
-        for iter_twitch_idx, iter_twitch_info in twitch_indices.items():
-            peak_amplitude = data_series[iter_twitch_idx]
-            prior_amplitude = data_series[iter_twitch_info[PRIOR_VALLEY_INDEX_UUID]]
-            subsequent_amplitude = data_series[iter_twitch_info[SUBSEQUENT_VALLEY_INDEX_UUID]]
-
-            amplitude_value = (
-                (peak_amplitude - prior_amplitude) + (peak_amplitude - subsequent_amplitude)
-            ) / 2
-
-            if rounded:
-                amplitude_value = round(amplitude_value, 0)
 
-            estimates_dict[iter_twitch_idx] = amplitude_value
+        for twitchPeakX, twitchData in coordinates.iterrows():
+            CONVERSION = 100000000
```

Comment:
What exactly is this conversion value? It's a magic a number with a name that doesn't not explain, but I'm assuming it has something to do with converting between µs and the interpolated timepoints. If so, it should use the `MICRO_TO_BASE_CONVERSION`